views/country_download.erb

```diff
@@ -62,21 +62,21 @@
         data on these politicians; great for spreadsheets, perfect for
         quick analysis.
       </p>
-      <ul class="grid-list grid-list--vertically-center" id="terms-<%= house[:slug].downcase %>">
-        <% house[:legislative_periods].each do |t| %>
-          <li id="term-<%= house[:slug].downcase %>-<%= t[:slug].downcase %>" style="margin-bottom:1.5em">
+      <ul class="grid-list grid-list--vertically-center" id="terms-<%= house.slug.downcase %>">
+        <% house.legislative_periods.each do |t| %>
+          <li id="term-<%= house.slug.downcase %>-<%= t.slug.downcase %>" style="margin-bottom:1.5em">
             <div class="avatar-unit">
               <span class="avatar"><i class="fa fa-university"></i></span>
-              <a href="<%= term_table_url(@page.country, house, t) %>"><h3><%= t[:name] %></h3></a>
+              <a href="<%= term_table_url(@page.country, house, t) %>"><h3><%= t.name %></h3></a>
               <p>
-                <% unless t[:start_date].to_s.empty? and t[:end_date].to_s.empty? %>
-                  <%= t[:start_date] %> - <%= t[:end_date] %>
+                <% unless t.start_date.to_s.empty? and t.end_date.to_s.empty? %>
+                 <%= t.start_date %> - <%=  t.end_date rescue '' %>
```

Comment:
If you cherry-pick https://github.com/everypolitician/viewer-sinatra/pull/14909/commits/77c2e3fc1e6bdf248b092afa8017acd4922f3821 you won't need to do this 'rescue'

Once these are proper date method calls, you can also simplify the previous line to `if t.start_date || t.end_date`